api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -72,11 +76,19 @@ message Bootstrap {
     // :ref:`LDS <arch_overview_dynamic_config_lds>` configuration source.
     core.v3.ConfigSource lds_config = 1;
 
+    // Resource locator for listener collection.
+    // [#not-implemented-hide:]
+    udpa.core.v1.ResourceLocator lds_resources_locator = 5;
```

Comment:
Is the plan to deprecate the parallel config source fields when these are implemented?

Reply:
I think it might be fine to keep these, as per the resolution algorithm.